### PR TITLE
Changes entry for Ripemd160 in 3.0

### PR DESCRIPTION
**Pull request by t8m**
The second commit belongs only to master branch. The first can be cherry picked to 3.0 (possibly with trivial conflict fixup).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -227,6 +223,12 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 [Migration guide]: https://github.com/openssl/openssl/tree/master/doc/man7/migration_guide.pod
 
+### Changes between 3.0.6 and 3.0.7 [?? ??? ????]
```
Comment (levitte) [inline CHANGES.md:226]: Any reason for `?? ??? ????` and not the usual `xx XXX xxxx`?

Comment (t8m) [reply CHANGES.md:226]: No reason. Just forgot that we do xx XXX xxxx instead. :grin:

**Review by tmshort — APPROVED** [review]
LGTM

**Comment by t8m** [comment]
Merged to master and cherry-picked the first commit to 3.0 branch after fixing trivial conflict.
Thank you for the reviews.

**Pull request closed without merging**